packages/zoe/test/unitTests/contracts/test-priceAggregator.js

```diff
@@ -1081,13 +1061,10 @@ test('storage', async t => {
       'mockChainStorageRoot.priceAggregator.ATOM-USD_price_feed',
     ),
     {
-      amountIn: { brand: { iface: 'Alleged: $ATOM brand' }, value: 1n },
-      amountOut: {
-        brand: { iface: 'Alleged: $USD brand' },
-        value: 1020n,
-      },
-      timer: { iface: 'Alleged: ManualTimer' },
-      timestamp: 1n,
+      amountIn: { brand: remotable('$ATOM brand'), value: 1n },
+      amountOut: { brand: remotable('$USD brand'), value: 1020n },
+      timer: remotable('ManualTimer'),
+      timestamp: { timerBrand: remotable('timerBrand'), absValue: 1n },
```

Comment:
this should probably use the `coerce` or `toAbs` -type helper, so we can keep the property names (`timerBrand` and `absValue`) semi-private to the implementation

Reply:
This makes sense, but not in a time crunch IMO (especially since there are other similar flavors). How would you feel about deferral?